projects/angular-sdk/src/components/cfar-offer-banner-large/cfar-offer-banner-large.component.ts

```diff
@@ -27,6 +27,7 @@ export class CfarOfferBannerLargeComponent extends GlobalEventComponent implemen
   @Input() uiVariant!: UiVariant;
   @Input() hasNoCoverageOption: boolean = true;
   @Input() hasWarningCoverageMessage: boolean = false;
+  @Input() env: string = "production";
```

Comment:
optional input and defaulting to production so that partners don't have to add this.